client/db/src/lib.rs

```diff
@@ -344,27 +352,13 @@ pub enum DatabaseSettingsSrc {
 	Custom(Arc<dyn Database<DbHash>>),
 }
 
-impl DatabaseSettingsSrc {
-	/// Return dabase path for databases that are on the disk.
-	pub fn path(&self) -> Option<&Path> {
```

Comment:
FYI: it's used in polkadot here https://github.com/paritytech/polkadot/blob/b900c5a868ff0b24751329f9f6452a3fe6681cc2/node/service/src/lib.rs#L720